packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -194,16 +190,16 @@ def routing_key(self) -> str | None:
 
 
 class RabbitResourceTrackingStartedMessage(RabbitResourceTrackingBaseMessage):
-    message_type: RabbitResourceTrackingMessageType = Field(
-        default=RabbitResourceTrackingMessageType.TRACKING_STARTED, const=True
-    )
+    message_type: Literal[
+        RabbitResourceTrackingMessageType.TRACKING_STARTED
+    ] = RabbitResourceTrackingMessageType.TRACKING_STARTED
 
-    wallet_id: WalletID | None
-    wallet_name: str | None
+    wallet_id: WalletID | None = None
+    wallet_name: str | None = None
```

Comment:
Q: Why are you adding defaults everywhere? (also in all other models in this PR).

Defaulting to `None` may lead to assumptions that these fields are optional, potentially introducing logic bugs if users don't properly check or handle None. In delicate topics such as billing-related code, an unintentional omission of `wallet_id` or `prciing_unit_..` can lead to issues like missed billing or incorrect account attribution.

By making the user explicitly handle these values (even though it is None), I think it is safer design in a critical area like billing.

Reply:
`bump-pydantic` (the "official" tool used to migrate the code) does it automatically, that's the reason why I have to check the changed files one by one...

Reply:
@matusdrobuliak66: https://github.com/pydantic/pydantic/issues/5481

Reply:
but I think there is something wrong here. We do not want them optional but required, and that they accept None as a value

Reply:
@sanderegg please, have a look at the documentation of previous Pydantic v1:
https://docs.pydantic.dev/1.10/usage/models/#required-optional-fields

Reply:
@sanderegg How I understand it, is that everywhere we thought it was required it was the whole time optional... so he is not really changing the current behavior

Reply:
@sanderegg @giancarloromeo but maybe the thing in reality we want is to put everywhere:
`wallet_id: WalletID | None = ...` -> this is probably how we intended to use it